soh/soh/Enhancements/randomizer/3drando/hints.cpp

```diff
@@ -820,19 +830,50 @@ void CreateGregRupeeHint() {
 
   Text temp1 = Text{
     "By the way, if you're interested, I saw the shiniest %gGreen Rupee%w somewhere in%g ",
-    "",
+    "Au fait, si ça t'intéresse, j'ai aperçu le plus éclatant des %gRubis Verts%w quelque part à %g",
     ""
   };
 
   Text temp2 = {
     "%w.^It's said to have %rmysterious powers%w...^But then, it could just be another regular rupee.&Oh well.",
-    "",
+    "%w. On dit qu'il possède des pouvoirs mystérieux... Mais bon, ça pourrait juste être un autre rubis ordinaire.",
```

Comment:
Looks like this is a translation for the Greg hint, was this intended to be in this PR? Is this AI generated or properly translated?

Reply:
This was properly translated by Hato, and was included here since the translation was missed on the original Greg PR.